starters/README.md

```diff
@@ -62,7 +62,7 @@ A starter is a simplified example to get up and running with Gatsby quickly and
 
 |                                 Name                                  | Demo                                                                                                | Description                                                   |
 | :-------------------------------------------------------------------: | --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
-|        [blog](https://github.com/gatsbyjs/gatsby-starter-blog)        | [gatsby-starter-blog-demo.netlify.com](https://gatsby-starter-blog-demo.netlify.com/)               | A starter for a blog powered by Gatsby and Markdown           |
+|        [blog](https://github.com/gatsbyjs/gatsby-starter-blog)        | [gatsby-starter-blog-demo.netlify.com](https://gatsby-starter-blog-demo.netlify.app/)               | A starter for a blog powered by Gatsby and Markdown           |
 |     [default](https://github.com/gatsbyjs/gatsby-starter-default)     | [gatsby-starter-default-demo.netlify.com](https://gatsby-starter-default-demo.netlify.com/)         | A simple starter to get up and developing quickly with Gatsby |
```

Comment:
```suggestion
|     [default](https://github.com/gatsbyjs/gatsby-starter-default)     | [gatsby-starter-default-demo.netlify.app](https://gatsby-starter-default-demo.netlify.app/)         | A simple starter to get up and developing quickly with Gatsby |
```

Reply:
Hi! I updated to update `*-demo.netlify.com` to `*-demo.netlify.app`